qa/rpc-tests/wallet_z_sendmany.py

```diff
@@ -23,7 +23,7 @@
 class WalletZSendmanyTest(BitcoinTestFramework):
     def setup_network(self, split=False):
         self.nodes = start_nodes(3, self.options.tmpdir, [[
-            nuparams(NU5_BRANCH_ID, 220),
+            nuparams(NU5_BRANCH_ID, 240),
```

Comment:
This changes by 20, but only 18 extra blocks are mined. I verified that this still works for the NU5 activation part of the test. Previously the test mined through to block 221, whereas now it mines to 239; in either case the next block would be NU5, so it counts as successfully activating.